### DISCUSSION TITLE
how to use it with laravel 8

### DISCUSSION THREAD

**Discussion by chuja3** [khaled-alshamaa/ar-php#8] [Q&A]
how to require class in controller in laravel 8 ?

`namespace App\Http\Controllers;

use Illuminate\Http\Request;
use PDF;

class PdfController extends Controller
{
    public function test(Request $request)
    {
        require_once '../vendor/ar-php/src/Arabic.php';
        $Arabic = new Arabic();`

Class "App\Http\Controllers\Arabic" not found

**Comment by khaled-alshamaa** [comment]
You can try this package for easy integration with Laravel (developed by @atmonshi):
https://github.com/atm-code/ar-php-laravel
